chrome-extension/tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "baseUrl": ".",
     "paths": {
       "@src/*": ["src/*"]
-    }
+    },
+    "types": ["node", "chrome"]
   },
-  "include": ["src", "utils", "vite.config.mts", "../node_modules/@types"]
+  "include": ["src", "utils", "vite.config.mts", "manifest.ts"]
```

Comment:
Was there an issue with referencing `../node_modules/@types`?

Reply:
I've removed it because line above i've placed `"types": ["node", "chrome"]`
But i think it's also good to include all times from `@types` i've back it :)